.github/workflows/haskell.yml

```diff
@@ -82,4 +82,5 @@ jobs:
       - run: cabal v2-build all --disable-optimization $CONFIG
       - run: cabal v2-test all --disable-optimization $CONFIG
       - run: cabal v2-haddock all $CONFIG
+        continue-on-error: true
```

Comment:
I _think_ this'll make it so that it only fails CI on doc errors, but I'm not sure.  Can someone else confirm it?

Reply:
That's my understanding too!

Reply:
I confirmed it because this CI failed :joy: